File: 3830-find-closest-person/3830-find-closest-person.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        dist1 = abs(z - x)
        dist2 = abs(z - y)
        return 1 if dist1 < dist2 else 2 if dist1 > dist2 else 0

File: 3830-find-closest-person/3830-find-closest-person.cpp
class Solution {
public:
    int findClosest(int x, int y, int z) {
        unsigned char dist1 = abs(z - x);
        unsigned char dist2 = abs(z - y);
        return dist1 < dist2 ? 1 : dist1 > dist2 ? 2 : 0;
    }
};